components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -1191,6 +1191,18 @@ func (ins *BackupFromBashtionImp) executeOnRemoteAndPoolStatus(commandString str
 		}()
 	}
 	sshUtil.getSSHConfig().hostIP = automateIps[0]
+
+	// If managed service and filesystem backup, info: "we don't have support for this configurations"
+	config, err := getExistingHAConfig()
+	if err != nil {
+		return status.Wrap(err, status.ConfigError, "unable to fetch HA config")
+	}
+
+	if config.Architecture.ConfigInitials.BackupConfig == "file_system" && isManagedServicesOn() {
```

Comment:
Use constant variables instead of strings.

Reply:
Please check if these conditions are true for:
- On prem deployment with AWS Managed DB
- AWS deployment

Reply:
Done!